### ISSUE TITLE
Inform users they need to allow push to receive tab

### ISSUE THREAD

**Issue by athomasmoz** [mozilla-mobile/firefox-ios#6566]
### Why/User Benefit/User Problem

As a user trying to send a tab to FF iOS, I want to be notified on FF iOS that I couldn't receive the tab because I haven't opted in to push notifications

### Acceptance Criteria (how do I know when I’m done?)

* Users informed when a tab is sent to FF iOS device, but not received because they are not opted in to push notifications
* Based on the information, user knows how to allow push notifications to make receive tab work

**Comment by ILikePlayingGames** [comment]
I agree. It’s nice to have the app tell me instead of having to search for the issue.
